### PR TITLE
PAN-20 Add displaying of ad unit page code

### PR DESCRIPTION
**Pull request by PawelPodkalicki**
In `src/app/publisher/dialogs/site-code-dialog/site-code-dialog.component.ts`
```
store: Store<AppState>
const userSubcrtiption = this.store.select('state', 'user', 'data')
```
 was changed to:
```
settingsService: SettingsService
this.settingsService.checkUserStatus()
```
basing on `withdraw-funds-dialog.component.ts`

### CODE REVIEW

File: src/app/publisher/dialogs/site-code-dialog/site-code-dialog.component.html [hunk 1]
```diff
@@ -30,6 +30,9 @@
         code"
       data-test="publisher-ad-code"
     >
+      &#x3C;!-- common code for all Ad Units, should be inserted into head section --&#x3E;<br/>
+      {{ data.codeCommon }}<br/><br/>
+      &#x3C;!-- specific code for this Ad Unit --&#x3E;<br/>
```
Comment (c3zi) [inline src/app/publisher/dialogs/site-code-dialog/site-code-dialog.component.html]: Could we format it in the same way like rest of the code here?
` &#x3C;!--` 
--> 
`<!--` 
??

File: src/app/models/site.model.ts [hunk 2]
```diff
@@ -17,6 +17,7 @@ interface Site {
     excludes: object;
   };
   adUnits?: AdUnit[];
+  pageCodeCommon?: string;
```
Comment (c3zi) [inline src/app/models/site.model.ts]: What is the difference between pageCodeCommon and pageCode?
I wonder if `scriptCode` is not better name?

File: src/app/publisher/dialogs/site-code-dialog/site-code-dialog.component.ts [hunk 3]
```diff
@@ -16,17 +17,23 @@ export class SiteCodeDialogComponent extends HandleSubscription implements OnIni
...
+    //     this.isEmailConfirmed = user.isEmailConfirmed;
+    //   });
+    // this.subscriptions.push(userSubscription);
+
+    this.settingsService.checkUserStatus()
       .subscribe((user: User) => {
         this.isEmailConfirmed = user.isEmailConfirmed;
       });
-    this.subscriptions.push(userSubcrtiption);
   }
```
Comment (c3zi) [inline src/app/publisher/dialogs/site-code-dialog/site-code-dialog.component.ts]: We can remove the commented code above. Can't we?

File: src/app/publisher/site-details/ad-units/ad-units.component.ts [hunk 4]
```diff
@@ -11,16 +11,17 @@ import { adSizesEnum } from 'models/enum/ad.enum';
...
   adSizesEnum = adSizesEnum;
 
   constructor(private dialog: MatDialog) { }
 
   openGetCodeDialog() {
     this.dialog.open(SiteCodeDialogComponent, {
       data: {
-        code: this.adUnit.code
+        code: this.adUnit.pageCode,
+        codeCommon: this.pageCodeCommon,
```
Comment (c3zi) [inline src/app/publisher/site-details/ad-units/ad-units.component.ts]: What is inside code and codeCommon?

File: src/app/publisher/dialogs/site-code-dialog/site-code-dialog.component.html (hunk 1 above)
Comment (PawelPodkalicki) [reply src/app/publisher/dialogs/site-code-dialog/site-code-dialog.component.html]: Comments mustb e to escaped to display user inside dialog.

File: src/app/publisher/dialogs/site-code-dialog/site-code-dialog.component.ts (hunk 3 above)
Comment (PawelPodkalicki) [reply src/app/publisher/dialogs/site-code-dialog/site-code-dialog.component.ts]: yes, we can;)

**Pull request merged**